pyeudiw/trust/__init__.py

```diff
@@ -30,8 +31,8 @@ def evaluation_method(self) -> bool:
         # method based on internal trust evaluetion property
         # TODO: implement the detect of x509 trust evaluation method here
         return self.federation
-
-    def _handle_chain(self):
+    
+    def _retrieve_anchor(self):
         _first_statement = unpad_jwt_payload(self.trust_chain[-1])
```

Comment:
what if the TA is X509 and not federation?

here we handle only federation, please generalize for different trust evaluation types